uptane/encoding/asn1_definitions.py

```diff
@@ -589,7 +589,7 @@ class VehicleVersionManifestSigned(univ.Sequence):
     namedtype.NamedType('vehicleIdentifier', Identifier().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 0))),
     namedtype.NamedType('primaryIdentifier', Identifier().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 1))),
     namedtype.NamedType('numberOfECUVersionManifests', Length().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 2))),
-    namedtype.NamedType('ecuVersionManifests', ECUVersionManifests().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 3))),
+    namedtype.NamedType('ecuVersionManifests', ECUVersionManifests().subtype(implicitTag=tag.Tag(tag.tagClassContext, tag.tagFormatSimple, 3))), # Should this be tagFormatConstructed?
```

Comment:
It's kinda complicated. I'm hoping for someone with more ASN.1 understanding than I have to discuss this with. http://luca.ntop.org/Teaching/Appunti/asn1.html is a start.

'ECUVersionManifests' is a List, and I'm not sure if List types should take tagFormatSimple or tagFormatConstructed when they're elements of another structure.

Oddly, there seems to be no difference in the DER produced whether I use tagFormatSimple or tagFormatConstructed, so I'm not sure I understand.